### PR TITLE
[pipelineX](fix) Fix nested loop join operator

### PR DESCRIPTION
**Pull request by Gabriel39**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Gabriel39** [comment]
run buildall

File: be/src/pipeline/pipeline_x/pipeline_x_task.cpp [hunk 1]
```diff
@@ -69,8 +69,8 @@ Status PipelineXTask::prepare(RuntimeState* state, const TPipelineInstanceParams
         auto& deps = get_downstream_dependency();
         std::vector<LocalSinkStateInfo> infos;
         for (auto& dep : deps) {
-            infos.push_back(LocalSinkStateInfo {_pipeline->pipeline_profile(),
-                                                local_params.sender_id, dep.get(), tsink});
+            infos.push_back(
```
Comment (BiteTheDDDDt) [inline be/src/pipeline/pipeline_x/pipeline_x_task.cpp:72]: emplace back

**Comment by Gabriel39** [comment]
run buildall

**Comment by Gabriel39** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**